includes/class-gth-temporary-post.php

```diff
@@ -3,6 +3,8 @@
 class Gth_Temporary_Post {
 	public $ID;
 	public $comments_open = 'open';
+	public $comment_status = 'open';
```

Comment:
In a local installation wit WordPress 6.1-alpha-53873 and GlotPress 4.0.0-alpha.1, it works without these 2 new lines. Are they necessary for some edge case, or can we remove it?

Reply:
I removed them, I first thought they might have been the root of the trouble.